content/docs/tasks/traffic-management/app-health-check/index.md

```diff
@@ -5,30 +5,40 @@ weight: 65
 keywords: [security,health-check]
 ---
 
-This task shows how to use [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/) for health checking of Istio services.
+This task shows how to use [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)
+for health checking of Istio services.
 
 There are three options for liveness and readiness probes in Kubernetes:
 
 1. Command
 1. HTTP request
 1. TCP request
 
-This task provides examples for the first two options with Istio mutual TLS enabled and disabled, respectively.
+This task provides examples for the first two options with Istio mutual TLS
+enabled and disabled, respectively.
 
 ## Before you begin
 
-* Understand [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/), Istio
-[authentication policy](/docs/concepts/security/#authentication-policies) and [mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication) concepts.
+* Understand [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/),
+Istio [authentication policy](/docs/concepts/security/#authentication-policies)
+and [mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication)
+concepts.
 
-* Have a Kubernetes cluster with Istio installed, without global mutual TLS enabled (meaning use `istio.yaml` as described in [installation steps](/docs/setup/kubernetes/quick-start/#installation-steps), or set `global.mtls.enabled` to false using [Helm](/docs/setup/kubernetes/helm-install/)).
+* Have a Kubernetes cluster with Istio installed, without global mutual TLS
```

Comment:
This prerequisite makes no sense. The text above explicitly states both scenarios are covered, mTLS enabled and disabled. I would remove this entire bullet.

Reply:
The examples below discuss mTLS disabled scenario first, that's why we need this prerequisite. And there is a section to enable mTLS at the end of "mTLS disabled scenario".